system/lib/libc/musl/src/math/ceil.c

```diff
@@ -28,4 +33,5 @@ double ceil(double x)
 	if (y < 0)
 		return x + y + 1;
 	return x + y;
+#endif
```

Comment:
This seems little odd to me.   Surely the compiler will normally convert `ceil` to `__builtin_ceil` at the callsite?

So this change only effect code built explicitly with `-fno-builtin`?    i.e it effect the libc itself, not moew user code?

It seems like there should be more precise way to compile libc that building the whole thing with `-fno-builtin`.